src/ingest-pipeline/airflow/dags/export_and_backup.py

```diff
@@ -0,0 +1,140 @@
+from pprint import pprint
+from datetime import datetime, timedelta
+import yaml
+from os.path import dirname, join
+from pathlib import Path
+import logging
+
+from airflow.configuration import conf as airflow_conf
+from airflow.operators.python import PythonOperator
+from airflow.exceptions import AirflowException
+
+import utils
+from utils import (
+    localized_assert_json_matches_schema as assert_json_matches_schema,
+    HMDAG,
+    get_queue_resource,
+)
+
+import export_and_backup.export_and_backup_plugin as export_and_backup_plugin
+
+PLUGIN_MAP_FILENAME = "export_and_backup_map.yml"
+
+# Following are defaults which can be overridden later on
+default_args = {
+    "owner": "hubmap",
+    "depends_on_past": False,
+    "start_date": datetime(2019, 1, 1),
+    "email": ["joel.welling@gmail.com"],
+    "email_on_failure": False,
+    "email_on_retry": False,
+    "retries": 1,
+    "retry_delay": timedelta(minutes=1),
+    "xcom_push": True,
+    "queue": get_queue_resource("validation_test"),
+}
+
+with HMDAG(
```

Comment:
Just to conserve the same code structure, please modify the spacing to be the same as all other DAGs (at least previous ones, I haven't looked the new plugin framework)

Reply:
Thanks Sunset! I see that my indentation for lines 36-40 doesn't match other DAGs. I think I also have some inappropriate double-line breaks. I have made these updates, in addition to switching to single-quotes per the other DAGs. Please let me know if you noticed anything else or if there are other changes you would like to request.

I see that we have some rules for linting using black in the repo's [pyproject.toml](https://github.com/hubmapconsortium/ingest-pipeline/blob/devel/pyproject.toml) file. If we want to enforce this style then I think we should update that file (and maybe even add some pre-commit hooks?). I'd be happy to work on figuring out some linting rules. What do you think?

Reply:
I honestly haven't checked that, but any improvement is welcome. If you feel comfortable enough working on it and see a good outcome of it, go for it and teach us.

Reply:
Sunset, @jpuerto-psc, @jswelling, are you all using isort & black? Trying to figure out if my configuration is busted, if we're not all using the same tools, or something else!